src/lib/Microsoft.Health.Fhir.Ingest/Template/CodeValueFhirTemplate.cs

```diff
@@ -9,6 +9,10 @@ namespace Microsoft.Health.Fhir.Ingest.Template
 {
     public class CodeValueFhirTemplate : FhirTemplate
     {
+#pragma warning disable CA2227
+        public virtual IList<FhirCodeableConcept> Category { get; set; }

```

Comment:
Should this be IList<FhirCode> like the "Codes" property below?

Reply:
Oh, I see, Codes is a singular CodeableConcept (list of codes), whereas Category is a list of CodeableConcepts: https://www.hl7.org/fhir/observation.html

Reply:
Yeap! Confusing at first but it makes sense.  It is something Will and I talked through today.  For codes it is singular codeable concept because it is still one concept but there could be different code systems (LOINC, SNOMED, etc).  Categories is different.  You could have two different concepts say something is a vital sign as well as a iomt.